node/src/types/deploy.rs

```diff
@@ -919,6 +919,23 @@ impl Deploy {
         Ok(())
     }
 
+    pub(crate) fn has_valid_hash(&self) -> Result<(), DeployConfigurationFailure> {
```

Comment:
Can you add a comment please?

Reply:
Fixed in [02645db](https://github.com/casper-network/casper-node/pull/3010/commits/02645db01331a35cc2ab337177741ca0ffae3f3f).